ext/pom.xml

```diff
@@ -2,10 +2,35 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
+    <name>Telekinesis</name>
+    <description>Telekinesis is a high-level client for Amazon Kinesis.</description>
+    <url>https://github.com/kickstarter/telekinesis</url>
     <groupId>com.kickstarter</groupId>
     <artifactId>telekinesis</artifactId>
     <version>3.1.0</version>
 
+    <licenses>
+       <license>
+           <name>TBD</name>
+           <url>TBD</url>
+           <distribution>repo</distribution>
```

Comment:
Thanks for pointing this out! Looks like we forgot our standard license stanza in the README. I just [added it](https://github.com/kickstarter/telekinesis#license).